src/apps/service/apps/job_listings.clj

```diff
@@ -76,7 +76,7 @@
   [rep-steps job]
   (remove-nil-vals
    (assoc (format-base-job job)
-          :external_ids     (vec (.getArray (:external_ids job)))
```

Comment:
The protocol extension defined in [apps.persistence.app-listings](https://github.com/cyverse-de/apps/blob/master/src/apps/persistence/app_listing.clj#L14-L21) tells `clojure.java.jdbc` how to convert an instance of `PgArray` to a Clojure vector, so the calls to `vec` and `.getArray` are no longer necessary.